src/components/ResponsiveContainer.js

```diff
@@ -1,9 +1,15 @@
 import Container from '../../node_modules/react-bootstrap/Container.js';
 import React from 'react';
+import MusicMarkdownNavbar from './MusicMarkdownNavbar.js';
+
+const RESPONSIVE_CONTAINER_KEY = 'app-container';
 
 function ResponsiveContainer(props) {
```

Comment:
We can convert this to the new style as well.

Reply:
Will do this with my next PR

Reply:
I lied, this passes state and needs props.